From 306d017e27abf96fec9fe908287f56b6e2f74dd6 Mon Sep 17 00:00:00 2001
From: shamim ali <ashamim@ami.com>
Date: Sat, 3 May 2025 06:35:58 -0400
Subject: [PATCH] removing sel callbacks update

this patch will remove logging callbacks to update the cache of ipmi-host
since using intel-ipmi-oem, this is unnecessary to update, it can be added
if using ipmi host implementation.

Signed-off-by: shamim ali <ashamim@ami.com>
---
 storagehandler.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/storagehandler.cpp b/storagehandler.cpp
index 4a97b47..b6f60da 100644
--- a/storagehandler.cpp
+++ b/storagehandler.cpp
@@ -872,7 +872,9 @@ ipmi::RspType<uint8_t,  // SDR version
 void register_netfn_storage_functions()
 {
     selCacheMapInitialized = false;
-    initSELCache();
+//    initSELCache();  //TODO: Currently using intel-ipmi-oem so initilization and registering callbacks
+    			// is not required for ipmi-host, it can be uncommented if ipmi-host implementation
+    			// will be used. 
     // Handlers with dbus-sdr handler implementation.
     // Do not register the hander if it dynamic sensors stack is used.
 
-- 
2.34.1

